changelog.txt

```diff
@@ -1,6 +1,6 @@
-=======
-Changes
-=======
+=========
+Changelog
+=========
```

Comment:
I'll allow it. ;)

Reply:
@csadorf This makes the name consistent with signac. 😉